samples/secure_tunneling/secure_tunnel/main.cpp

```diff
@@ -333,7 +325,7 @@ int main(int argc, char *argv[])
          */
         secureTunnel = SecureTunnelBuilder(
                            Aws::Crt::g_allocator,
-                           bootstrap,
+                           *apiHandle.GetOrCreateStaticDefaultClientBootstrap(),
```

Comment:
can we update SecureTunnelBuilder with a path that doesn't require bootstrap?  Unfortunately, there's no builder so it's constructor combinatorics which is unfortunate.